Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -2571,6 +2571,24 @@ void LocalFrameView::scrollElementToRect(const Element& element, const IntRect&
     setScrollPosition(IntPoint(bounds.x() - centeringOffsetX - rect.x(), bounds.y() - centeringOffsetY - rect.y()));
 }
 
+void LocalFrameView::setScrollOffsetWithOptions(std::optional<int> x, std::optional<int> y, const ScrollPositionChangeOptions& options)
+{
+    if (x && y) {
+        setScrollOffsetWithOptions({ *x, *y }, options);
+        return;
+    }
```

Comment:
Optional, but the rest of the function would do this for identical amount of work, right?

Reply:
it avoids a call to `scrollOffsetFromPosition(scrollPosition())` in this path